RaisingTheBar.asciidoc

```diff
@@ -1,34 +1,70 @@
 
 *	100% of major and critical bugs will be resolved
 *	50% of minor bugs will be resolved
 *	Cosmetic issues will be fixed on a best endeavour basis
 
 === Test Plans, Suites & Cases
 
-A test plan is a group of test cases. We recommend a Test Plan is made up from a number of Test Suites (one for each Feature). You then create a test case per user story.
+A test plan is a collection of test cases. We recommend a Test Plan to be made up of a number of Test Suites (each contains a group of Test Cases related to 1 Feature). We therefore need to ensure that each-and-every Story is therefore cross-referenced and checked with 1 or more test cases.
+
+A test case is the steps that validate and verify the quality of a product within the user story. Each step should have a defined expected result. A tester can thus validate the test case step by step, and create a bug or enhancement at the designated step that caused the failure. Along with a screenshot (always include a screenshot and any other relevant information), logs, and used test data, this should give developers the resources necessary to troubleshoot the majority of issues.
+
+=== Managing Test Data
+
+Real-world data plays a crucial role in testing. It is vital to prepare and manage realistic and comprehensive *test data* sets to simulate production scenarios. Ensure that data covers all edge cases, security aspects, and performance metrics so that the final product behaves reliably in a real-world context.
 
-A test case is the steps that validate a user story. Each step should have an expected result. A tester can thus validate the test case step by step, and create a bug at the designated step that caused the failure. Along with a screenshot (always include a screenshot and any other relevant information), this should give developers the resources necessary to troubleshoot the majority of issues.
+=== Continuous Feedback and Improvement
+
+Effective *defect management* and continuous feedback loops between QA and development teams ensure that issues are addressed promptly. So keep in mind that a streamlined process for tracking defects, setting priorities, and reporting results is vital. Regular reviews of key metrics such as defect density and mean time to resolution (MTTR) are really nice tools for assessing the health of the project and making informed decisions for continuous improvement.
+
+=== AI in QA
+
+As QA continues to evolve, *AI and machine learning* are becoming key players in advancing the field. From automating test case generation to predictive analytics for identifying high-risk areas in code, AI can augment traditional QA practices. Exploring these technologies will help future-proof testing processes and drive even more robust software quality.
 
 === Static code analysis
```

Comment:
I think we need a main "Types of Testing" section, and then under that we can run through the different types of testing. We could probably re-use the text in the test strategy template that explains the different types of testing. For example, we need "Chaos Monkey" testing in there :-)

Reply:
Nice idea, done.